components/post_view/post_body_additional_content/post_body_additional_content.jsx

```diff
@@ -107,11 +111,16 @@ export default class PostBodyAdditionalContent extends React.PureComponent {
 
     UNSAFE_componentWillReceiveProps(nextProps) { // eslint-disable-line camelcase
         if (nextProps.post.message !== this.props.post.message) {
+            const embedMetadata = nextProps.post.metadata.embeds && nextProps.post.metadata.embeds[0];
```

Comment:
Oh i thought post metadata obj always exists but combined system messages does not. I would be updating this